checkOnExternalSystemsAd.ps1

```diff
@@ -0,0 +1,184 @@
+#########################################################################
+# HelloID-Conn-Prov-Target-Blacklist-Check-On-External-Systems-AD-SQL
+#########################################################################
+
+#Todo: Exclude person.externalId from search
+
+# Initialize default values
+$success = $false # Set to false at start, at the end, only when no error occurs it is set to true
+$auditLogs = [System.Collections.Generic.List[PSCustomObject]]::new()
+$NonUniqueFields = [System.Collections.Generic.List[PSCustomObject]]::new()
+
+# The entitlementContext contains the configuration
+# - configuration: The configuration that is set in the Custom PowerShell configuration
+$eRef = $entitlementContext | ConvertFrom-Json
+$c = $eRef.configuration
+
+# The account object contains the account mapping that is configured
+$a = $account | ConvertFrom-Json
+
+# Set debug logging
+switch ($($c.isDebug)) {
```

Comment:
Debug toggle in AD zie je volgens mij bijna nooit? write-verbose aanpassen naar write-information?

Reply:
Aangepast